Applications/DataExplorer/DataView/MeshMapping2DDialog.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "MeshMapping2DDialog.h"
+#include "OGSError.h"
+#include "StrictDoubleValidator.h"
+
+#include <QSettings>
+#include <QFileDialog>
+
+MeshMapping2DDialog::MeshMapping2DDialog(QDialog* parent)
+: QDialog(parent)
+{
+	setupUi(this);
+
+	StrictDoubleValidator* no_data_validator = new StrictDoubleValidator(this);
```

Comment:
At this point I'm not sure who is going to delete the no_data_validator. The QLineEdit has empty dtor, so this looks like a memory leak to me.